app/screens/auth/UnlockWallet.tsx

```diff
@@ -135,15 +171,32 @@ const UnlockWallet = ({ history, location }: AuthRouterParams) => {
     };
   }, [history, location]);
 
-  const getDropDownData = () =>
-    walletFiles.length === 0
-      ? [{ label: 'NO WALLET FILES FOUND', isDisabled: true }]
-      : walletFiles.map(({ meta }) => ({
-          label: meta.displayName,
-          description: `CREATED: ${formatISOAsUS(meta.created)}, NET ID: ${
-            meta.genesisID
-          }`,
-        }));
+  const dropDownData = useMemo(() => {
+    if (walletFiles.length === 0) {
+      return [{ label: 'NO WALLET FILES FOUND', isDisabled: true }];
+    }
+
+    return walletFiles.map(({ meta, path }) => {
+      const networkName = networksMapName[meta.genesisID] ?? 'NOT SELECTED';
+      return {
+        label: meta.displayName,
+        description: `CREATED: ${convertISOToDate(
+          meta.created
+        )} | NETWORK: ${networkName}`,
+        endAdornment: (
+          <>
+            {copiedPath === path ? (
+              <CopiedBanner>Wallet path copied</CopiedBanner>
+            ) : (
+              <CopiedButtonWrapper>
+                <CopyButton secondary value={path} onClick={setCopiedPath} />
```

Comment:
I believe "Reveal in Finder" will be more handy for a common user.
You can use the folder icon for it.
Also, you won't need additional state and animation.